### PR TITLE
Login and registration page

### PR DESCRIPTION
**Pull request by anderCM**
Login page and registration page should be built with Devise

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  devise_for :users
+  root :to => redirect('foods')
+  resources :users, only: [:index, :show]
+  resources :foods, only: [:index, :show, :new, :create, :destroy]
```
Comment (JakariaJishan) [inline config/routes.rb:7]: it's really nice you added routes.

File: app/views/devise/confirmations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (JakariaJishan) [inline app/views/devise/confirmations/new.html.erb:1]: nice to have devise you added

**Review by JakariaJishan — APPROVED** [review]
### Hi @anderCM ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


FragiFragiskosGIF (2)


## Highlights

✔️ All models are created according wireframes
✔️ Relationship `one-to-many` and `many-to-many` added correctly
✔️ Validations correctly added to models

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (anderCM) [reply config/routes.rb:7]: Thanks @JakariaJishan

File: app/views/devise/confirmations/new.html.erb (hunk 2 above)
Comment (anderCM) [reply app/views/devise/confirmations/new.html.erb:1]: Thanks @JakariaJishan  😁

**Pull request merged**